### ISSUE TITLE
CropGrowthChance Incompability with Fabric Seasons

### ISSUE THREAD

**Issue by abrossimow** [augustsaintfreytag/minecraft-crop-growth-modifier#1]
When using Fabric Seasons CropGrowthChance is ignored.

Not sure, which Mixin is colliding here. Maybe:
https://github.com/lucaargolo/fabric-seasons/blob/1.20/src/main/java/io/github/lucaargolo/seasons/utils/FertilizableUtil.java

Thank you very much for this mod, it works great and is very well made.

**Comment by augustsaintfreytag** [comment]
Haven't looked into it in detail yet but at first glance, I'm suspecting they're calling `grow` functions on the entity directly while my mod intervenes right before that, where the game would call `grow` down the chain, like random tick and growth apply (e.g. bonemeal).

I'm guessing Seasons doesn't add its own configuration options?

A more extreme fix to this could be to add a high priority mixing for `grow` calls themselves but that might have some side effects. Could be an option, though.

**Comment by abrossimow** [comment]
> I'm guessing Seasons doesn't add its own configuration options?
Right. There is an option to disable Season's influence on bone mealing but that just nulls an if-bool in the logic itself, and does not disable their mixin.

**Comment by AugustSaintFreytag** [comment]
I'm preparing for the 2.0.0 release. The mod should now support all vanilla and modded blocks and interferes at the random tick stage. Bone meal/item apply features, as well as growth acceleration have been removed for now. If there's still issues with seasons, you can of course open a new issue.
